src/components/SearchResult.js

```diff
@@ -3,22 +3,25 @@ import "./SearchResultsList.css"
 import HighlightedContent from "./HighlightedContent";
 
 // Search Result Component 
-export const SearchResult = ({ item, query })=> {
+export const SearchResult = ({ item, query }) => {
 
-    useEffect(() => {
-        
-        //console.log('This is the term' + term);
-
-    })
-
-    return(
-        <a key={item.id} href={item.url} target="_blank" rel="noopener noreferrer" style={{
-            textDecoration: 'none', color: 'inherit'}}>
-            <div className="search-result">
-                    <p className="title" style={{color: "blue"}}>{item.title}</p>
-                    <p className="url">{item.url}</p>
-                    <HighlightedContent content={item.content} query={query}/>
+    return (
+        <div className="search-result">
+            <a
+                key={item.id}
+                href={item.url}
+                target="_blank"
+                rel="noopener noreferrer"
```

Comment:
why?

Reply:
The href={item.url} attribute is used to specify the URL that the link should navigate to when clicked. 

The target="_blank" attribute sets the behaviour of the link to open in a new tab or window when clicked. 

The rel="noopener noreferrer" attribute is used as a security measure when using target="_blank". It helps protect against potential security vulnerabilities, such as reverse tab nabbing.

Reply:
ah, didn't know about tab nabbing